dm/dm/config/subtask.go

```diff
@@ -25,12 +25,12 @@ import (
 	bf "github.com/pingcap/tidb-tools/pkg/binlog-filter"
 	"github.com/pingcap/tidb-tools/pkg/column-mapping"
 	"github.com/pingcap/tidb-tools/pkg/filter"
-	router "github.com/pingcap/tidb-tools/pkg/table-router"
```

Comment:
just a question: why not pull a request to tidb-tools?
maybe i miss some offline talk infomation😂